### DISCUSSION TITLE
Simplify UI of Model Details page

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#26] [Ideas]
## Briant Mock-up
![image](https://user-images.githubusercontent.com/11724394/210155777-a4412161-2bb9-46a2-9540-811c00c2a89d.png)

## Latest Mock-up
![Details](https://user-images.githubusercontent.com/607609/205224439-b98288d9-dc69-42f1-8e07-5efb433ea50f.png)

## Old Mock-up
![image](https://user-images.githubusercontent.com/607609/201821822-f1c6f417-1289-4174-b053-98c8e8d95293.png)

**Comment by manuelurenah** [comment]
The discussion rework is live! 🚀 Users are now able to comment on models and reply to these comments and reviews as well. Here are a couple of screenshots of how it looks like

![SCR-20221207-ipf](https://user-images.githubusercontent.com/12631159/206252393-30f245ff-f52f-4707-855e-bf21d73036c1.jpeg)
![SCR-20221207-irb](https://user-images.githubusercontent.com/12631159/206252407-34c43c63-ab10-4b98-94c3-9400244f0ddb.jpeg)
![SCR-20221207-irr](https://user-images.githubusercontent.com/12631159/206252414-d9e6fd38-a5f0-4da9-91e7-80a320544036.jpeg)
![SCR-20221207-is8](https://user-images.githubusercontent.com/12631159/206252418-f1a6c69d-3b63-419e-915a-658b4129ef7c.jpeg)
